app/views/datasets/_contact.html.erb

```diff
@@ -1,9 +1,11 @@
 <%
-  if contact_email_is_email?(dataset)
-    mail = mail_to(contact_email_for(dataset), nil, {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name, class: 'govuk_link'})
-  else
-    mail = link_to(contact_email_for(dataset), contact_email_for(dataset), {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name, class: 'govuk_link'})
-  end
+  default_name = "Contact #{@dataset.organisation.title} regarding this dataset"
```

Comment:
Would be great is we move this hardcoded copy into `locales`/translation file – they allow for parameters to be passed in, to deal with title.